src/styles/components/navigation.scss

```diff
@@ -0,0 +1,123 @@
+.navigation {
+    $navbar-height: 60px;
+    $subnav-height: 50px;
+    $subnav-btn-width: 130px;
+    $arrow-size: 25px;
+    position: absolute;
```

Comment:
is it possible to make without absolute?
How will the components which are under the navigation look like?